pkg/cmd/server/api/types.go

```diff
@@ -626,6 +627,10 @@ type MasterNetworkConfig struct {
 	IngressIPNetworkCIDR string
 }
 
+type ClusterNetworkEntry struct {
+	ClusterNetworkCIDR string
```

Comment:
Maybe just `CIDR`? You've already got `ClusterNetwork` in the struct name and field name...

Also, if the plan is to eventually have per-network HostSubnetLength values, I think we should add that here now; if we don't support different values yet, you can just add code to make sure every ClusterNetworkEntry has the same value.

(Also, if we're tweaking HostSubnetLength handling, we might want to consider renaming it and flipping the math around; the fact that it counts bits from the end of the IP address rather than the start is confusing. (Eg, HostSubnetLength of 9 means each node gets a  /23 subnet (`32 - 9 = 23`). I think it would be clearer for most users if the config option was "23" rather than "9".